utils/utils.c

```diff
@@ -392,3 +392,81 @@ char * strjoin(char *left, char *right, char *delim)
 	strcpy(new + len - rlen - 1, right);
 	return new;
 }
+
+#define QUOTE '\''
+#define DQUOTE '"'
+
+static int setargs(char *args, char **argv)
```

Comment:
Oh, I meant a header commit which is right above the function signature and describes arguments and return value.  Especially for `argv`...  But I think it should be for `parse_cmdline` and `free_parsed_cmdline()`.